fuzzers/baby_fuzzer_minimizing/src/main.rs

```diff
@@ -0,0 +1,142 @@
+use std::path::PathBuf;
+#[cfg(windows)]
+use std::ptr::write_volatile;
+
+use libafl::prelude::*;
+
+/// Coverage map with explicit assignments due to the lack of instrumentation
+static mut SIGNALS: [u8; 16] = [0; 16];
+
+/// Assign a signal to the signals map
+fn signals_set(idx: usize) {
+    unsafe { SIGNALS[idx] = 1 };
+}
+
+#[allow(clippy::similar_names)]
+pub fn main() -> Result<(), Error> {
+    // The closure that we want to fuzz
+    let mut harness = |input: &BytesInput| {
+        let target = input.target_bytes();
+        let buf = target.as_slice();
+        signals_set(0);
+        if !buf.is_empty() && buf[0] == b'a' {
+            signals_set(1);
+            if buf.len() > 1 && buf[1] == b'b' {
+                signals_set(2);
+                if buf.len() > 2 && buf[2] == b'c' {
+                    return ExitKind::Crash;
+                }
+            }
+        }
+        ExitKind::Ok
+    };
+
+    // Create an observation channel using the signals map
+    let observer =
+        unsafe { StdMapObserver::new_from_ptr("signals", SIGNALS.as_mut_ptr(), SIGNALS.len()) };
+
+    let factory = MapEqualityFactory::new_from_observer(&observer);
+
+    // Feedback to rate the interestingness of an input
+    let mut feedback = MaxMapFeedback::new(&observer);
+
+    // A feedback to choose if an input is a solution or not
+    let mut objective = CrashFeedback::new();
+
+    // The Monitor trait define how the fuzzer stats are displayed to the user
+    let mon = SimpleMonitor::new(|s| println!("{}", s));
+
+    let mut mgr = SimpleEventManager::new(mon);
+
+    let corpus_dir = PathBuf::from("./corpus");
+    let solution_dir = PathBuf::from("./solutions");
+
+    // create a State from scratch
+    let mut state = StdState::new(
+        // RNG
+        StdRand::with_seed(current_nanos()),
+        // Corpus that will be evolved, we keep it in memory for performance
+        OnDiskCorpus::new(&corpus_dir).unwrap(),
```

Comment:
is a `OnDiskCorpus` kept in memory?

Reply:
No, that'd need a CachedOnDiskCorpus or a InMemoryCorpus

Reply:
So the comment is outdated here